### ISSUE TITLE
`coq.gr->string` duplicates the identifier

### ISSUE THREAD

**Issue by CohenCyril** [LPCIC/coq-elpi#100]
```coq
Elpi Command debug.
Elpi Query debug lp:{{
  coq.locate "bool" GR,
  coq.gr->string GR ID.
}}.
```
outputs
```
Query assignments:
  GR = indt «bool»
  ID = Coq.Init.Datatypes.bool.bool
```

**Comment by gares** [comment]
Not 100% sure of recalling why I did it like that, but the code does it on purpose :-)
Unfortunately I did not bind mutual inductives yet, so I can't test this easily, but my guess is that
it would print `...even.even` and `...even.odd` for example, since the first part comes from the block of mutual inductives while the second from the name of the single inductive (in the inductive block).
It prints `....bool.bool.true` for constructors as well...

`coq.gr->id` probably does what you want.

I leave this open since I don't know what to do. If I remove the duplication I'm afraid that it would print the same for both even and odd, that is not something you want.
